src/main/res/values/strings.xml

```diff
@@ -41,6 +41,14 @@
 
     <string name="sendingVote">Sending vote</string>
 
+    <string name="send">Send</string>
+
+    <string name="cancel">Cancel</string>
+
+    <string name="fillFieldsVoteDialog">Please select one of the three voting options before sending the vote</string>
+
+    <string name="addMessage">Write here a comment to help the speaker know what is your opinion on the talk</string>
```

Comment:
As @MaureenElsberry pointed out on the ios version the message should read: "Leave a comment on your opinion of this talk to help out the speaker."